.github/workflows/windows-build-and-test.yaml

```diff
@@ -27,9 +27,9 @@ jobs:
       id: build_type
       run: |
         if [[ "${{ github.event_name }}" == "pull_request" ]]; then
-          echo "::set-output name=build_type::['Debug']"
+          echo "build_type=['Debug']" >>$GITHUB_OUTPUT
         else
-          echo "::set-output name=build_type::['Debug','Release']"
+          echo "build_type=['Debug','Release']" >>$GITHUB_OUTPUT
```

Comment:
There is also is a reference to`set-output` in Line 184

Reply:
Yes, but that does not work with WSL bash shell and causes errors: https://github.com/timescale/timescaledb/actions/runs/3289818410

Reply:
I was also unable to perform this with `wsl-bash`. However, you could move the file test to a separate step and do it with PowerShell (`pwsh`). This works well for me.

```
name: Windows test
on:
  workflow_dispatch:
  
jobs:
  build:
    runs-on: windows-2022
    
    steps:
    - name: Setup WSL
      uses: Vampire/setup-wsl@v1
         
    - name: Create file
      shell: wsl-bash {0}
      run: |
         touch regression.log
         
    - name: Test file exists
      shell: pwsh
      id: collectlogs
      run: |
         if (Test-Path -Path regression.log) {
            echo "regression_diff=true" >> ${env:GITHUB_OUTPUT}
         }
         
    - name: Save regression diffs
      if: steps.collectlogs.outputs.regression_diff == 'true'
      shell: wsl-bash {0}
      run: |
         echo "Yes"
```

Reply:
Or just create a python script. The biggest problem is, however, that since it complains about `GITHUB_OUTPUT` not being set, then does the file exist at all and is the environment variable set at all?

Reply:
Let's test.

Reply:
I have tested both shells. The problem seems to only affect the `wsl-bash`. In `pwsh`, the environment variables are set correctly. In my test workflow, the `if: steps.collectlogs.outputs.regression_diff == 'true'` statement was evaluated correctly based on the `pwsh` output.

Reply:
Switched to using Python instead, which is easier to maintain.